tests/.bazelrc

```diff
@@ -1,5 +1,3 @@
-common --config=tests
```

Comment:
`.bazelrc_shared` still contains this line:
```
build:tests --disk_cache=../.bazel_cache
```

If delete ` common --config=tests`, won't that option no longer take effect?

Reply:
Yep. I missed that and will deal with this.